editors/sc-ide/widgets/audio_status_box.cpp

```diff
@@ -137,8 +137,10 @@ void AudioStatusBox::onServerRunningChanged(bool running, const QString&, int, b
 }
 
 void AudioStatusBox::wheelEvent(QWheelEvent* event) {
-    if (!event->angleDelta().isNull()) {
-        if (event->angleDelta().y() > 0 || event->inverted())
+    // If Alt is pressed, Qt swaps scroll axis: undo it because we use alt to change scale
+    double delta = event->modifiers().testFlag(Qt::AltModifier) ? event->angleDelta().x() : event->angleDelta().y();
+    if (delta) {
+        if (delta > 0 || event->inverted())
```

Comment:
Ah, this is incorrect - now the volume _only_ goes up. Shouldn't it be something like `(delta > 0) * event->inverted().not`? (not the right syntax probably)